### PR TITLE
fixed translations for fraktal hard

### PR DESCRIPTION
**Pull request by Akurosia**

### CODE REVIEW

File: ui/raidboss/data/04-sb/dungeon/fractal_continuum_hard.js [hunk 1]
```diff
@@ -268,4 +292,190 @@
...
+        'Light Pillar': 'リヒト・ゾイレ',
+        'Northern Star': 'ノーザンスター',
+        'Primordial Aether': 'プライマルエーテル',
+        'Ratzon': 'ラツォン',
+        'Southern Star': 'サザンスター',
+      },
+    },
+    {
+      'locale': 'cn',
+      'missingTranslations': true,
```
Comment (ShadyWhite) [inline ui/raidboss/data/04-sb/dungeon/fractal_continuum_hard.js:408]:
```suggestion
```

File: ui/raidboss/data/04-sb/dungeon/fractal_continuum_hard.js [hunk 2]
```diff
@@ -268,4 +292,190 @@
...
+      'replaceSync': {
+        'Exhibit level VIII': '第八展示区',
+        'Motherbit': '浮游炮主板',
+        'Prototype Bit': '原型浮游炮',
```
Comment (ShadyWhite) [inline ui/raidboss/data/04-sb/dungeon/fractal_continuum_hard.js:412]:
```suggestion
        'Prototype Bit': '原型浮游炮',
        'The Genesis Engine': '启动试验室',
```

File: ui/raidboss/data/04-sb/dungeon/fractal_continuum_hard.js [hunk 3]
```diff
@@ -268,4 +292,190 @@
...
+        'The Genesis Engine': '启动试验室',
```
Comment (ShadyWhite) [inline ui/raidboss/data/04-sb/dungeon/fractal_continuum_hard.js:413]:
```suggestion
        'The Genesis Engine': '启动试验室',
        'This humanoid prototype can perfectly replicate': '我们研究出了如何利用斗神力量的方法！',
```

File: ui/raidboss/data/04-sb/dungeon/fractal_continuum_hard.js [hunk 4]
```diff
@@ -268,4 +292,190 @@
...
+        'The Ultima Beast': '究极神兽',
+        'The Ultima Warrior': '究极战士',
+        'The reality augmentation bay': '现实增强室',
```
Comment (ShadyWhite) [inline ui/raidboss/data/04-sb/dungeon/fractal_continuum_hard.js:416]:
```suggestion
        'The reality augmentation bay': '现实增强室',
        'Utilizing our data on Sophia': '利用斗神的力量去征服斗神！',
```

File: ui/raidboss/data/04-sb/dungeon/fractal_continuum_hard.js [hunk 5]
```diff
@@ -268,4 +292,190 @@
...
+        'Vocal Guidance System': '语音向导',
+      },
+      'replaceText': {
+        'Aether Bend': '以太曲折',
+        'Aetherochemical Laser': '魔科学激光',
+        'Aetheroplasm': '以太爆雷',
```
Comment (ShadyWhite) [inline ui/raidboss/data/04-sb/dungeon/fractal_continuum_hard.js:422]:
```suggestion
        '(?<! )Aetheroplasm': '以太爆雷',
```

File: ui/raidboss/data/04-sb/dungeon/fractal_continuum_hard.js [hunk 6]
```diff
@@ -268,4 +292,190 @@
...
+        'Death Spin': '死亡回旋',
+        'Demi Ultima': '亚究极',
+        'Diffractive Laser': '扩散射线',
+        'Dischordant Cleansing': '不平衡之罚',
+        'Electrochemical Transfer': '雷力供给',
+        'Flare Star': '耀星',
+        'Forborn Beast': '后天野兽',
+        'Infinite Fire': '炎之刻印',
+        'Infinite Ice': '冰之刻印',
+        'Light Pillar': '光柱',
```
Comment (ShadyWhite) [inline ui/raidboss/data/04-sb/dungeon/fractal_continuum_hard.js:436]:
```suggestion
        'Light Pillar': '光柱',
        'Mass Aetheroplasm': '大型以太爆雷',
```

**Review by ShadyWhite — CHANGES_REQUESTED** [review]
Add missing CN traslations in timelineReplace.

**Comment by Akurosia** [comment]
@ShadyWhite could you double check if i did not mess up any CN translations (as i did them in batch with the other languages as well)

**Comment by MaikoTan** [comment]
Hey Akurosia, I can confirm that `replaceText/Sync` of CN/JA were completed. Thanks!

And I recommand to use the *translate timeline* feature of my [VSCode extension](https://marketplace.visualstudio.com/items?itemName=MaikoTan.cactbot-highlight) when editing replacements. You can easily find out if there are untranslated lines.

**Comment by Akurosia** [comment]
@MaikoTan i use my own scripts currently and technicly all lines were translated 
but things like `1248.5 "--sync--" sync / 00:0044:Vocal Guidance System:This humanoid prototype can perfectly replicate/ window 10,10 jump 1100.0` had originally only the part `Vocal Guidance System` translated

**Comment by MaikoTan** [comment]
> @MaikoTan i use my own scripts currently and technicly all lines were translated
> but things like `1248.5 "--sync--" sync / 00:0044:Vocal Guidance System:This humanoid prototype can perfectly replicate/ window 10,10 jump 1100.0` had originally only the part `Vocal Guidance System` translated

Yeah, I think it is not easy to check partial translation like this. Unless the timeline itself can be generated by some pre-defined script.
But you just added it at the recent commit, didn't you?

**Comment by Akurosia** [comment]
yes, several things, once i notice they happen in a timeline translation, i do a manually fix than.
Currently i am on reworking my script to not miss these things and in worst case just creates a translation to much (e.g. `.*is no longer sealed` is part of the common translations)
but currently i have a lot of rules to capture now most parts automaticly

**Pull request merged**